recipes/simhash-py/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "simhash-py" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "c83e20cfac873ba9067c564135c293db635cbd7372f126e9bcfb0fabf9fd0234" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/27/42/fc4203f6881e98a744765f7911d783bbd67d5c85706e7e66d1c06e6c6534/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please replace with this [url]( https://github.com/conda-forge/staged-recipes/blob/9fdcfeaf287eed4f4b62d681162b3b6392ca27a6/recipes/example/meta.yaml#L18 ) from the example.